x-team/GamesHQ-FE
# Add prettier, husky & eslint

## Fixes:
https://x-team-internal.atlassian.net/browse/CORE-2612

## How to test
- Run `npm i --legacy-peer-deps`
- Try changing a file and check if it's being formatted using the pre commit hook.

## Screenshots
<img width="849" alt="Screenshot 2024-10-17 at 5 17 21 PM" src="https://github.com/user-attachments/assets/263ea81b-1e9a-4cb8-84fa-078736ceb525">
